src/scss/_color-use-cases.scss

```diff
@@ -1,8 +1,19 @@
-@include oColorsSetColor('jade-black-70', oColorsMix('jade', 'black', 70));
-
-@include oColorsSetUseCase('o-cookie-message-alternative', 'border', 'jade');
-@include oColorsSetUseCase('o-cookie-message-alternative-button', 'text', 'white');
-@include oColorsSetUseCase('o-cookie-message-alternative-button', 'background', 'jade');
-@include oColorsSetUseCase('o-cookie-message-alternative-button', 'hover', 'jade-black-70');
-@include oColorsSetUseCase('o-cookie-message-alternative-link', 'text', 'black');
-@include oColorsSetUseCase('o-cookie-message-alternative-link', 'background', 'white');
+@include oColorsSetColor('o-cookie-message/jade-black-70', oColorsMix('jade', 'black', 70));
+
+@include oColorsSetUseCase('o-cookie-message/alternative', (
```

Comment:
This is correct but we don't need to share these colours with any other component so it'd be better to make these private Sass variables

Reply:
oh that's good news, i didn't understand why these were in a usecase when it seemed less complex (and easier to read!) to use a variable. very glad to do that